src/main/kotlin/no/nav/klage/kaka/api/view/RawDataExport.kt

```diff
@@ -0,0 +1,71 @@
+package no.nav.klage.kaka.api.view
+
+import java.util.*
+
+data class AnonymizedVurdering(
+    //should be fine with saksdataId b/c we validate access when accessing saksdata
```

Comment:
Hm... Jeg ser poenget i det du skriver, men jeg er samtidig ikke 100% sikker på om vi bør gjøre det for lett for folk. Men jeg klarer ikke å si at dette ikke er greit, så ok.